src/Client.ts

```diff
@@ -43,22 +48,20 @@ import {
   SearchResponse,
   search,
 } from "./api-endpoints"
-
-import got, {
-  Got,
-  Options as GotOptions,
-  Headers as GotHeaders,
-  Agents as GotAgents,
-} from "got"
+import crossFetch from "cross-fetch"
```

Comment:
I suggest importing `cross-fetch` as a polyfill `import 'cross-fetch/polyfill'` instead of importing as an instance, then getting it from the global scope because it will be the environment `fetch` if it exists or be the crossFetch polyfill if it doesn't.

This way it may have better chance to work on non Node environments like Deno.

Reply:
I don’t think a library should force a specific polyfill on library consumers if it can be avoided.